catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntryExtractor.java

```diff
@@ -50,7 +50,15 @@
      * @throws GeoEntryExtractionException if an error occurs while extracting GeoNames entries from
      *                                     the resource
      */
-    void getGeoEntriesStreaming(String resource, ExtractionCallback extractionCallback);
+    void pushGeoEntriesToExtractionCallback(String resource, ExtractionCallback extractionCallback);
```

Comment:
I think this would be better as:
     void pushGeoEntriesToExtractionCallback(URI resource, ExtractionCallback extractionCallback);